_includes/hero-body.html

```diff
@@ -11,7 +11,7 @@
                             <h1 class="is-hidden-mobile">{{ site.title }}</h1>
                             <h1 class="is-hidden-mobile">{{ site.wg_title }}</h1>
 
-                            <img src="/images/muscat-sm.png" class="mb-3 is-hidden-tablet">
+                            <img src="/images/logo.png" class="mb-3 is-hidden-tablet">
```

Comment:
Has this been tested on a smaller screen? Do we want to hide the logo on tablet view, as the class name would suggest?

Reply:
It is the same as now - same logo for all screen sizes

Reply:
It is a bulma class for not showing it on tablet. It should not be removed because otherwise the logo will appear twice

Reply:
Or maybe it can. I will have to test it

Reply:
I just resized the page on guidelines-stage and at no size did the muscat logo disappear, so I'm not sure what that class does?